packages/@react-aria/visually-hidden/docs/VisuallyHidden.mdx

```diff
@@ -59,3 +59,17 @@ let {visuallyHiddenProps} = useVisuallyHidden();
 
 <div {...visuallyHiddenProps}>I am hidden</div>
 ```
+
+## Gotchas
```

Comment:
better section title? use inline alert?

Reply:
Caveats?

Reply:
I think Gotchas is fine here, no need for an inline alert imo